lib/components/viewers/live-stop-times.tsx

```diff
@@ -219,6 +219,7 @@ class LiveStopTimes extends Component<Props, State> {
                   this.renderDay(homeTimezone, time.day, now)}
                 <PatternRow
                   homeTimezone={homeTimezone}
+                  key={pattern.id}
```

Comment:
Should the key be applied to the surrounding fragment instead?

Reply:
I thought so too but this is what stopped the console warnings